<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="mainList">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>341</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="upButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>350</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
image: url(:/buttons/upButton.svg);
background-color: #ffffff;
border-radius: 20px;
}

QPushButton:pressed {
	background-color: #ffd300;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="downButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>470</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
image: url(:/buttons/downButton.svg);
background-color: #ffffff;
border-radius: 20px;
}

QPushButton:pressed {
	background-color: #ffd300;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="leftButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>410</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
image: url(:/buttons/rightButton.svg);
background-color: #ffffff;
border-radius: 20px;
}

QPushButton:pressed {
	background-color: #ffd300;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="rightButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>410</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
image: url(:/buttons/leftButton.svg);
background-color: #ffffff;
border-radius: 20px;
}

QPushButton:pressed {
	background-color: #ffd300;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>350</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
image: url(:/buttons/backButton.svg);
background-color: #ffffff;
border-radius: 20px;
}

QPushButton:pressed {
	background-color: #ffd300;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="menuButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>350</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
image: url(:/buttons/menuButton.svg);
background-color: #ffffff;
border-radius: 20px;
}

QPushButton:pressed {
	background-color: #ffd300;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="okButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>410</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
image: url(:/buttons/okButton.svg);
background-color: #ffffff;
border-radius: 20px;
}

QPushButton:pressed {
	background-color: #ffd300;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="menuLabel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="powerButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>530</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
image: url(:/buttons/powerButton.svg);
background-color: #ffffff;
border-radius: 20px;
}

QPushButton:pressed {
	background-color: #ffd300;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGraphicsView" name="powerOffView">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>341</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <zorder>powerOffView</zorder>
   <zorder>mainList</zorder>
   <zorder>upButton</zorder>
   <zorder>downButton</zorder>
   <zorder>leftButton</zorder>
   <zorder>rightButton</zorder>
   <zorder>backButton</zorder>
   <zorder>menuButton</zorder>
   <zorder>okButton</zorder>
   <zorder>menuLabel</zorder>
   <zorder>powerButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>636</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
